### PR TITLE
Issue #107

### PR DESCRIPTION
**Pull request by marcelo-maciel**
Issue #107

### CODE REVIEW

File: src/FlaUI.Core/Input/Wait.cs [hunk 1]
```diff
@@ -16,12 +16,12 @@ public static class Wait
         /// Waits for a generic time which was found to be sufficient to allow
         /// input (mouse, keyboard, ...) do be processed
         /// </summary>
-        public static void UntilInputIsProcessed()
+        public static void UntilInputIsProcessed(int millisecondsToWait = 100)
```
[the remark this answers is not in the document]
Comment (marcelo-maciel) [reply src/FlaUI.Core/Input/Wait.cs]: Yes! I will do the change.

File: src/FlaUI.Core/Input/Wait.cs [hunk 2]
```diff
@@ -16,12 +16,16 @@ public static class Wait
...
-        public static void UntilInputIsProcessed()
+        /// <param name="waitTimeout">An optional timeout. If null is passed, the timeout is infinite.</param>
+        public static void UntilInputIsProcessed(TimeSpan? waitTimeout = null)
         {
             // Let the thread some time to process the system's hardware input queue.
             // For details see this post: http://blogs.msdn.com/b/oldnewthing/archive/2014/02/13/10499047.aspx
             // TODO: Should this be configurable?
-            Thread.Sleep(100);
+            
+            var waitTime = (waitTimeout ?? TimeSpan.FromMilliseconds(-1)).TotalMilliseconds;
```
Comment (Roemer) [inline src/FlaUI.Core/Input/Wait.cs]: Default should be 100 ms not -1 :)

Comment (BogdanLivadariu) [reply src/FlaUI.Core/Input/Wait.cs]: https://msdn.microsoft.com/en-us/library/system.threading.timeout.infinitetimespan(v=vs.110).aspx

Comment (Roemer) [reply src/FlaUI.Core/Input/Wait.cs]: So in other words, if you don't pass a value it will wait forever. Definitely not what we want as default behavior.

**Comment by Roemer** [comment]
Replaced with PR #134.

**Pull request closed without merging**